README.md

```diff
@@ -211,6 +211,7 @@ could also build them yourself!_
 * [$10 WiFi RGB Bulb](https://community.home-assistant.io/t/how-to-inexpensive-10-us-wifi-rgb-bulb-that-works-with-home-assistant/14735?u=frenck) - In inexpensive RGB bulb that works on WiFi.
 * [433mhz/IR Bidirectional Gateway](https://community.home-assistant.io/t/433mhz-infrared-ir-to-and-from-mqtt-on-esp8266/6779?u=frenck) - Bidirectional with IR and 433mhz using ESP8266 and MQTT.
 * [esp8266MQTTBlinds](https://community.home-assistant.io/t/esp8266-window-blinds-mqtt/14863?u=frenck) - Automate your window blinds using an ESP8266, a servo and MQTT.
+* [Hackster.io](https://www.hackster.io/home-assistant?f=1#_=_) - The Hackster channel for Home-Assistant with multiple DIY projects.
```

Comment:
We should avoid using Home Assistant in the description, since its kinda implied...
Give the fact it links to a specific channel and not just the Hackster.io website, I'd suggest we update this as follows:

- `Home Assistant's Hackster.io`
- `A Hackerster channel with multiple DIY projects.`

Or something similar...

What do you think @robmarkcole?